build.gradle

```diff
@@ -316,7 +258,6 @@ task setupIntellijStaticAnalysis {
     dependsOn copyIntelliJProjectPluginsSettings
     finalizedBy copyStylelintConfiguration
```

Comment:
@wkurniawan07 At that time of doing this, I felt it was the best way but I guess I must have been abusing `Task#finalizedBy`. Is that what you mean?

The reason for not using `Task#dependsOn` is to enforce ordering, I'm aware of`Task#mustRunAfter` but it is still incubating and pretty new.

However, I suppose I am wrong about trying to avoid new incubating features because I think Gradle probably mark so many features as incubating probably so they don't have to freeze their APIs and keep their velocity up.

While the code theoretically won't break after changing to `Task#dependsOn`, the ordering is no longer clear, with that said can we add `Task#mustRunAfter`? I do think I enforced the wrong ordering though, I wanted all required configurations to be copied first before setting up the plugins. 😅

Reply:
> I wanted all required configurations to be copied first before setting up the plugins

`copyStylelintConfiguration.dependsOn copyIntelliJProjectPluginsSettings`?

Reply:
Yea but that logically looks strange to me.

Because then `setupIntellijStaticAnalysis ` will only depend on `copyStylelintConfiguration` which is not true? Unless we rename `copyStylelintConfiguration`?

Reply:
@whipermr5 I think you're looking on `mustRunAfter'

Reply:
Oops; I've edited the order. But was looking for a solution other than `mustRunAfter` since @LiHaoTan mentioned that it's an incubating feature.

Reply:
The problem with that argument is even `Task#finalizedBy` is also an incubating feature...

Reply:
@whipermr5 well `finalizedBy` is incubating still too, but it had been around longer lol, so what I'm observing is that they probably leave things incubating for a very long time unless they can confirm.